Formula/access.rb

```diff
@@ -0,0 +1,33 @@
+class Access < Formula
+  desc "Easiest way to request and grant access without leaving your terminal"
+  homepage "https://indent.com"
+  url "https://github.com/indentapis/access.git",
+      tag:      "v0.10.9",
+      revision: "19955980a9fb76ef294ea19829e0479c37e81898"
+  license "Apache-2.0"
+  head "https://github.com/indentapis/access.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.GitVersion=#{Utils.git_head}
+    ]
+
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/access"
+
+    # Install shell completions
+    generate_completions_from_executable(bin/"access", "completion")
+  end
+
+  test do
+    test_file = testpath/"access.yaml"
+    test_file.write("")
+    Dir.chdir(testpath) do
+      system bin/"access", "config", "set", "space", "some-space"
+    end
```

Comment:
`testpath` is already the working directory.
```suggestion
    system bin/"access", "config", "set", "space", "some-space"
```